### PR TITLE
chore(*): CI initialization

### PR DESCRIPTION
**Pull request by russellwheatley**
Repository setup for CI integration.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
```
Comment (Salakar) [inline .github/workflows/release.yaml:6]:
```suggestion
      - main
```

File: .github/workflows/scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+#     3: Release Git tag
+create_github_release() {
+  local response=''
+  local release_name=$1
+  local release_body=$2
+  local release_tag=$3
+
+  local body='{
+    "tag_name": "%s",
+    "target_commitish": "master",
```
Comment (Salakar) [inline .github/workflows/scripts/release.sh:30]:
```suggestion
    "target_commitish": "main",
```

File: .github/workflows/scripts/release.sh [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+update_github_release() {
+  local response=''
+  local release_name=$1
+  local release_body=$2
+  local release_tag=$3
+  local release_id=$4
+
+  local body='{
+	  "tag_name": "%s",
+	  "target_commitish": "master",
```
Comment (Salakar) [inline .github/workflows/scripts/release.sh:72]:
```suggestion
	  "target_commitish": "main",
```

**Review by Salakar — CHANGES_REQUESTED** [review]

**Comment by Salakar** [comment]
Some notes on this if we want the release scripts running on this repository also (cc @jhuleatt)

 - Default branch would need changing to `next` from the current `main` to match `firebase/extensions` release process.
 	- Merges from `next` to `main`  would trigger the release, PRs however target  the default branch `next` only.
 - `GITHUB_TOKEN` secret would need to be set on this repository.

Other than that LGTM 👍

**Comment by jhuleatt** [comment]
Thanks @russellwheatley and @Salakar! 

> * Default branch on this repository would need changing to `next` from the current `main` to match `firebase/extensions` release process.
>   * Merges from `next` to `main`  would trigger the release, PRs however target  the default branch `next` only.

Done

> * `GITHUB_TOKEN` secret would need to be set on this repository.

This is [set automatically](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret), so we should be all set there.

**Review by Salakar — APPROVED** [review]

**Comment by russellwheatley** [comment]
Merge this PR first to fix formatting error: https://github.com/FirebaseExtended/experimental-extensions/pull/9

**Comment by jhuleatt** [comment]
Thank you both!

**Review by jhuleatt — APPROVED** [review]

**Pull request merged**